packages/suite/src/components/suite/layouts/SuiteLayout/Sidebar/QuickActions/Update/UpdateStatusActionBarIcon.tsx

```diff
@@ -195,33 +192,30 @@ export const UpdateStatusActionBarIcon = ({
         }
     };
 
-    const getTooltip = (): Partial<ManagedTooltipProps> => {
-        if (discoveryInProgress) {
-            return {
-                cursor: 'not-allowed',
-                content: <Translation id="TR_UNAVAILABLE_WHILE_LOADING" />,
-            };
-        }
+    const anyUpdateInfoAvailable = isDesktopSuite || !discoveryInProgress;
```

Comment:
now, displaying the tooltip and icon if one of those is available, a pattern matching would be better here though.

Reply:
will do pattern matching :)